create-redirects.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+# Author: Austin Kregel <github@kbco.me>
```

Comment:
Can you put some kind of description about what this script actually does?

Reply:
Sure can :) 

The other one didn't, so I didn't know if there was a pattern you wanted me to follow.